mmdet/ops/merge_cells.py

```diff
@@ -0,0 +1,125 @@
+from abc import abstractmethod
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+
+class BaseMergeCell(nn.Module):
+
+    def __init__(self,
+                 channels=256,
+                 binary_op_type='sum',
+                 with_conv=True,
+                 conv_cfg=dict(),
+                 norm_cfg=None,
+                 order=('act', 'conv', 'norm'),
+                 with_input_conv_x=False,
+                 with_input_conv_y=False,
```

Comment:
Use unified names for the inputs. If they are `x1`, `x2` in the forward method, we can name these two arguments `with_input1_conv` and `with_input2_conv`.